examples/wasm-demo-cairo1/README.md

```diff
@@ -0,0 +1,50 @@
+# Demo of `cairo-vm` on WebAssembly
+
+While cairo-vm is compatible with WebAssembly, it doesn't implement any bindings to it.
+Instead, create a new WebAssembly crate with cairo-vm as a dependency and implement the required functionality there.
+
+Since mimalloc is not automatically compilable to WebAssembly, the cairo-vm dependency should disable the default features, which will in turn disable mimalloc.
+
+A working example is provided in this repository.
+
+## Dependencies
+
+To compile and run the example you need:
+
+- a Cairo 0 compiler
```

Comment:
Check the readme, there are lot of things you copied from the Cairo 0 wasm demo and don't apply here.